GraphStructure/Vertex.java

```diff
@@ -0,0 +1,16 @@
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.Iterator;
+
+public class Vertex{
+	public Integer id;
+	public HashSet<Integer> adjvertex;
```

Comment:
I would suggest you use Edge object as well. In the future, we may need to add extra attributes in the edge. Here, it should be a HashMap<id, Edge>. Pay attention to the naming. There are a couple of points. (1) "adjvertex", but we have a set of vertices. (2) "adj" and "vertex" are two independent words, so concatenate them in camel capital: adjVertices. Sticking to this naming convension, check out the style guide please.